docs/appendix.rst

```diff
@@ -143,6 +143,22 @@ This documentation uses a few special terms to refer to Python types:
       backslash-escaping any double quotes in INSTANCENAME, and putting it into
       double quotes.
 
+   default connection
+      A connection definition in the :ref:`--server general option` that is used
+      by pywbemcli as the :term:`current connection` if pywbemcli is started without
+      any connection definition (no :ref:`--server general option`,
+      :ref:`--mock-server general option`, or :ref:`--name general option`).
+      A connection definition in the :term:`connections file` becomes the
+      current connection on pywbemcli startup if it is defined using the
+      :ref:`connection select command`.
+
+   current connection
+      The connection definition in pywbemcli that is currently active; the
+      target WBEM server for pywbemcli commands.  The current server is defined
+      either upon startup (:ref:`--server general option`,
+      :ref:`--mock-server general option`, or :ref:`--name general option` or
+      :term:`default connection`) or with the :ref:`connection select command`.
```

Comment:
It is **always** defined on startup: If none of these three general options is specified, the default connection is used. That needs to be clarified. The select command **changes** the current connection.

Reply:
Not quite.  There may be no default. In that case, pywbemcli is started with no connection defined.  In that case, commands like connection will work but commands like ;class enumerate` will fail because no server is defined.  In that case, the select would create a current connection rather than change an existing one.

Rewrote that one.  DONE